tests/02_generic_input/field_python_test.py

```diff
@@ -0,0 +1,25 @@
+import flowpy
+
+class PyFieldAnisotropy(flowpy.PythonFieldBase):
+    pass
+
+    def used_fields(self):
+        field_list = ["X"]
+        return field_list
+    
+    def __call__(self):
+        diag = (0.5+2*z if (z < 0.25) else 1)
+        return ( diag, 0, 0, 0, diag, 0, 0, 0, diag )
+
+
+class PyFieldWaterSourceDensity(flowpy.PythonFieldBase):
```

Comment:
- Add dependency on other field.
- Add comments to make it a bit of tutorial how to implement FieldPython class.